tools/manifest_files/version_map.yaml

```diff
@@ -1,8 +1,7 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: supported-upgrade-paths
-  namespace: default 
+  name: supported-versions-map
```

Comment:
Is there a reason to remove the namespace. Will it work if we upgrade from 4.3 to 5.1?

Reply:
No reason, I missed it I have changed it back.